docs/dev/agent_integration.md

```diff
@@ -452,33 +428,19 @@ sudo datadog-agent integration install -w /path/to/wheel.whl
   ```
 </details>
 
-## Review the checklist to publishing your integration
-
-After you've created your Agent-based integration, refer to this list to make sure your integration contains all the required files and validations:
-
-- A `README.md` file with the correct format and contents.
-- A battery of tests verifying metrics collection.
-- A `metadata.csv` file listing all of the collected metrics.
-- A complete `manifest.json` file.
-- If the integration collects Service Checks, the `service_checks.json` must be complete as well.
-
-Before you open a pull request, run the following command to catch any problems with your integration:
-```
-ddev validate all <INTEGRATION_NAME>
-```
+## Populate your tile and publish your integration
 
-After you've created your pull request, automatic checks run to verify that your pull request is in good shape and contains all the required content to be updated.
+Once you have created your Agent-based integration, populate the remaining [required assets][31] that  appear on your integration tile. For more information on populating these assets and best practices, see [Create a Tile][20].
 
-## Further Reading
+Finally, open a pull request with your code on the [`integrations-extras`][21] or [`marketplace`][22] repository. After you've created your pull request, automatic checks run to verify that your pull request is in good shape and contains all the required content to be updated.
```

Comment:
```suggestion
```

I don't think this paragraph is useful. I think creating a tile should be the last thing the user sees on this page - that page has much better instructions for opening the pull request - and the steps they have to perform before they open the PR - like the validation step.